Modix/Handlers/MessageLogHandler.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Discord;
+using Discord.WebSocket;
+using Discord.Rest;
+using Modix.Services.BehaviourConfiguration;
+using Serilog;
+
+namespace Modix.Handlers
+{
+    public class MessageLogHandler
```

Comment:
Would you consider making this an `IBehavior` down in `Modix.Services` since

A) This is core business logic, not just part of the bot frontent
B) It'll be one less thing to rework later when `ModixBotHooks` gets replaced.

Reply:
Certainly; I was following the same pattern as the existing InviteLinkHandler, but it makes sense to refactor those into Modix.Services.

(I am now realizing we have two different concepts named very similarly - `Behavior`s and `Behaviour`s - we'll need to refactor that a bit...)

Reply:
That was my suggestion when Inzanit first introduced `BehaviourConfiguration`. On its own, the name makes sense, but in context, it implies that it's closely related to `IBehavior`, which isn't necessarily the case.

I was thinking `ApplicationConfiguration` or `ConfigurationSetting`.

Long term, I'd much like all configuration-related stuff to run through the `ConfigurationActions` table so any changes to bot config get logged. I'll probably just open an issue soon to discuss re-implementing the feature a bit.